build-system/build.conf.js

```diff
@@ -26,9 +29,69 @@ const defaultPlugins = [
   require.resolve('./babel-plugins/babel-plugin-transform-version-call'),
 ];
 
+/**
+ * @return {Array<string|Object>} the minify-replace plugin options that can be
+ * pushed into the babel plugins array
+ */
+function getReplacePlugin() {
+  /**
+   * @param {string} defineStr the define flag to parse
+   * @return {!Object} replacement options used by minify-replace plugin
+   */
+  function createReplacement(defineStr) {
+    const strSplit = defineStr.split('=');
+    const identifierName = strSplit[0];
+    // if no value is defined, set to true
+    const value = strSplit.length > 1 ? strSplit[1] === 'true' : true;
+
+    return {
+      identifierName,
+      replacement: {
+        type: 'booleanLiteral',
+        value,
+      },
+    };
+  }
+
+  const replacements = [];
+  const defineFlag = argv.define;
```

Comment:
did we settle on `define` as the flag name?

Reply:
nope the name isn't settled, any suggestions?

Reply:
discussed offline, settled on `defineExperimentConstant`